quickstart_lists/core_full_x86_64-linux_stable

```diff
@@ -0,0 +1,709 @@
+core/7zip
+core/acbuild
+core/acl
+core/airlock
+core/alex
+core/alsa-lib
+core/ansible
+core/ant
+core/apr
+core/apr-util
+core/artifactory
+core/artifactory-pro
+core/asciinema
+core/aspcud
+core/aspnet-sample
+core/atk
+core/at-spi2-atk
+core/at-spi2-core
+core/attr
+core/autoconf
+core/autogen
+core/automake
+core/aws-cli
+core/azure-cli
+core/bash
+core/bash-completion
+core/bash-static
+core/bats
+core/bazel
+core/bc
+core/bdwgc
+core/bdwgc7
+core/bdwgc8
+core/benchmark
+core/bind
+core/binutils
+core/bison
+core/bison2
+core/boost
+core/boost159
+core/boringssl
+core/builder-admin
```

Comment:
All the "builder" packages should be removed from this list - the ones in core are obsolete